angular/projects/spark-angular/src/lib/components/inputs/sprk-radio-item/sprk-radio-item.component.spec.ts

```diff
@@ -51,21 +51,21 @@ class TestComponent3 {}
     </sprk-radio-item>
   `,
 })
-class TestComponent4 {}
+class Test4Component {}
 
 describe('SprkRadioItemComponent', () => {
-  let component1: TestComponent1;
-  let component2: TestComponent2;
-  let component3: TestComponent3;
-  let component4: TestComponent4;
+  let component1: Test1Component;
```

Comment:
Oh yikes, was TestComponent4 causing a dup identifier issue?

Reply:
yeah, was throwing a tslint error, i guess components have to end with 'Component' in test files